StudyBuddy/Controllers/ProfileController.cs

```diff
@@ -31,13 +35,13 @@ public IActionResult DisplayProfiles()
 
     public IActionResult UserId(string id)
     {
-        UserId parseUserId= (UserId)Guid.Parse(id);
+        UserId parseUserId = (UserId)Guid.Parse(id);
 
         IUser? user = _userManager.GetUserById(parseUserId);
 
         if (user != null)
         {
-            return View("DisplayProfiles", new List<IUser?>() {user});
+            return View("DisplayProfiles", new List<IUser?>() { user });
```

Comment:
Creating a new list everytime only for one user?

Reply:
We were reusing the same view for single and all profiles, maybe fix this in another branch, creating the full profile view, adding more traits?

Reply:
Agree